xiaogpt/config.py

```diff
@@ -76,6 +76,8 @@ class Config:
     enable_edge_tts: bool = False
     edge_tts_voice: str = "zh-CN-XiaoxiaoNeural"
     gpt_options: dict[str, Any] = field(default_factory=dict)
+    cookie_file: str = "./cookies.json"
+    conversation_style: str = "creative"
```

Comment:
Again, no need to config them specially, as they only work with a specific bot type. It is not ideal. Just remove them and users can customize them with `gpt_options` anyway.